test/instance.ts

```diff
@@ -498,6 +532,34 @@ describe('Instance', () => {
       instance.get(assert.ifError);
     });
 
+    it('should accept `fields` string', done => {
+      const fieldNames = 'name';
+      instance.request = config => {
+        assert.deepStrictEqual(config.reqOpts, {
+          fieldMask: {
+            paths: arrify(fieldNames),
+          },
+          name: instance.formattedName_,
+        });
+        done();
+      };
+      instance.get({fieldNames}, assert.ifError);
+    });
+
+    it('should accept `fields` array', done => {
+      const fieldNames = ['name', 'labels'];
+      instance.request = config => {
+        assert.deepStrictEqual(config.reqOpts, {
+          fieldMask: {
+            paths: fieldNames,
+          },
+          name: instance.formattedName_,
+        });
+        done();
+      };
+      instance.get({fieldNames}, assert.ifError);
+    });
+
```

Comment:
can we add unit test case for nested type as mention in below docs:
https://github.com/googleapis/nodejs-spanner/blob/1c0cf27b6c502fe7cc03f9b4c9191d746761b8b6/protos/google/protobuf/field_mask.proto#L42-L174

Reply:
I believe there is no case of nested types for `FieldMask` in `getMetadata()` for `Instance`
Available values for `fieldNames` array are:
`'name'`, `'displayName'`, `'endpointUris'`, `'labels'`, `'config'`, `'nodeCount'`, `'state'`